src/Button/Button.scss

```diff
@@ -322,61 +379,75 @@ input[type="button"] {
   --pgn-btn-hover-color: #{$btn-brand-hover-color};
   --pgn-btn-hover-bg: #{$btn-brand-hover-bg};
   --pgn-btn-hover-border-color: #{$btn-brand-hover-border-color};
-  --pgn-btn-disabled-color: #{$btn-brand-color};
-  --pgn-btn-disabled-bg: #{$btn-brand-bg};
-  --pgn-btn-disabled-border-color: #{$btn-brand-border-color};
+  --pgn-btn-disabled-color: #{$btn-brand-disabled-color};
+  --pgn-btn-disabled-bg: #{$btn-brand-disabled-bg};
+  --pgn-btn-disabled-border-color: #{$btn-brand-disabled-border-color};
   --pgn-btn-active-color: #{$btn-brand-active-color};
   --pgn-btn-active-bg: #{$btn-brand-active-bg};
   --pgn-btn-active-border-color: #{$btn-brand-active-border-color};
+  --pgn-btn-focus-outline-color: #{$btn-brand-focus-outline-color};
+  --pgn-btn-focus-color: #{$btn-brand-focus-color};
+  --pgn-btn-focus-border-color: #{$btn-brand-focus-border-color};
+  --pgn-btn-focus-bg: #{$btn-brand-focus-bg};
 
   box-shadow: $btn-box-shadow;
```

Comment:
[curious] I see `box-shadow: $btn-box-shadow;` is on many of these `.btn-*` class names, but not all of them. Should it be? Even though I believe the variable is currently essentially set for having no box shadow, if it _did_ have a box shadow, would it be applying equally across all the button variants or only a subset of them?

Reply:
Declaring the `box-shadow: $btn-box-shadow;` is necessary for some buttons on the documentation site. This applies exclusively to the Edxorg theme.

**For example:**
- Openedx theme
![image](https://user-images.githubusercontent.com/93188219/219070196-9e3eea3b-fd4d-431b-8ac5-1bb965f657e0.png)

- Edxorg theme
![image](https://user-images.githubusercontent.com/93188219/219070426-96f53cdd-d966-4c09-943d-548f6a315b0b.png)